pyproject.toml

```diff
@@ -0,0 +1,207 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+
+[project]
+name = "braingeneers"
+authors = [
+  { name = "Braingeneers", email = "me@example.com" },
+]
+description = "Braingeneers Python utilities"
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
```

Comment:
Update to BSD License classifier, you can find it at: https://pypi.org/classifiers/